site/docs/v1/tech/identity-providers/_samlv2-integration-details-fields.adoc

```diff
@@ -3,6 +3,8 @@
 An Assertion Consumer Service (or ACS) is the location at a service provider that accepts messages to establish a session based on the assertion.
 +
 By default the redirect URI will be the first Authorized redirect URI found in the FusionAuth Application OAuth configuration. To specify a redirect URI send the authorized URI in the `RelayState` parameter. Support for using the `RelayState` in this manner is supported in FusionAuth version greater than or equal to `1.41.0`. In prior versions, append a query parameter named `redirect_uri` to the ACS in the IdP configuration.
++
+Note: For IdP initiated logins in order to receive a refresh token the query parameter `?scope=offline_access` will need to be appended to the url. When logging into the FusionAuth Admin application this parameter is included by default and does not need to be supplied.
```

Comment:
This doesn't read quite right.. maybe needs some commas? 

Suggestion:

> Note: To receive a refresh token when completing the OAuth2 workflow when using an IdP initiated login, ensure you request the `offline_access` scope.  To request a scope, add the following query parameter to your configured ACS in your IdP, `?scope=offline_access`. When logging into the FusionAuth admin UI, this step is optional as the `offline_scope` will be implicitly added.

Reply:
My first pass actually read very similar to this but I tried to cut down the words for more brevity.